cpp/src/Slice/Parser.cpp

```diff
@@ -5133,13 +4724,6 @@ Slice::Const::containedType() const
     return ContainedTypeConstant;
 }
 
-bool
-Slice::Const::uses(const ContainedPtr& contained) const
-{
-    ContainedPtr contained2 = dynamic_pointer_cast<Contained>(_type);
-    return (contained2 && contained2 == contained);
-}
```

Comment:
this code was wrong with the upgrade to shared_ptr, and still everything worked! noticed it is not used anymore.